staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
I'm not sure if we should update it to `2019`,  because it basically porting from legacy code.